prepared_test.go

```diff
@@ -4,10 +4,18 @@
 
 package websocket
 
+/* TODO: Test fails after changing from math/rand to crypto/rand in conn.go since we are unable to provide the same seed to the random number generator.
```

Comment:
Quick que, we'll be revisiting this at a later point? Or, we have already got this cleared and the comment section is just a stale?

Reply:
Once this pull request merges I will be opening an issue on this repository about getting this test (or something similar) up and running again as a high priority.

Reply:
This has been fixed and the tests re-enabled.